test/Twig/Tests/Extension/IntlTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+require_once __DIR__ . '/../../../../lib/Twig/Extensions/Extension/Intl.php';
+
+/**
+ * @author Remy Gazelot <r.gazelot@gmail.com>
+ */
+class Twig_Tests_Extension_IntlTest extends PHPUnit_Framework_TestCase
+{
+    public static function setUpBeforeClass()
+    {
+        if (!class_exists('Twig_Extensions_Extension_Intl')) {
+            self::markTestSkipped('Unable to find class Twig_Extensions_Extension_Intl.');
+        }
+    }
+
+    public function testLocalizedDateFilterWithDateTimeZone()
+    {
+        if (PHP_VERSION_ID < 50500) {
+            $this->markTestSkipped('Only in PHP 5.5+ IntlDateFormatter allows to use DateTimeZone objects.');
+        }
+
+        $date = twig_localized_date_filter(
+            $this->env = $this->getMock('Twig_Environment'),
+            new DateTime('2015-01-01T00:00:00', new DateTimeZone('UTC')),
+            'short',
+            'long',
+            null,
+            '+02:00'
```

Comment:
@stof I have not used a DateTimeZone object here because of instantiate a DateTimeZone with ``+02:00`` has been fixed in PHP 5.5.9 http://3v4l.org/SDMWN